src/lib/github/getCheckedComments.ts

```diff
@@ -0,0 +1,16 @@
+import { getBasicRepoProps } from "./utils";
+
+import { Context } from "./types";
+
+export default async function getCheckedComments (context: Context): Promise<string[]> {
+  const octokit = context.github;
+  const { owner, repo } = getBasicRepoProps (context);
+  const fields = {
+      owner,
+      repo,
+      labels: ["GH-TODO-BOT"],
```

Comment:
This label value is now used in more than one place. i.e in `addIssuesToRepo` module and here.
I would suggest we make it a `const` variable within `addIssuesToRepo` that we export out and import in for use with this module.